public/css/desktop-style.css

```diff
@@ -1,176 +1,177 @@
 #main {
-    display: block;
```

Comment:
This un necessay spaces.... not should be removed...
Spaces 4 are used in all css files.

Reply:
Done